.env.example

```diff
@@ -1,16 +1,24 @@
-# Gemfile
-GEM_PG_VERSION=1.2.2
+# Database
+DB_HOST=postgis
+DB_NAME=gtt
+DB_USER=gtt
+DB_PASS=gtt
+DB_PORT=5432
 
-# docker-compose.yml - postgres
-POSTGRES_USER=gtt
-POSTGRES_PASSWORD=gtt
-POSTGRES_DB=gtt
-
-# config/configuration.yml
-SMTP_ENABLE_STARTTLS_AUTO=true # (or false)
-SMTP_ADDRESS=smtp.gmail.com
-SMTP_PORT=587
-SMTP_DOMAIN=smtp.gmail.com
-SMTP_AUTHENTICATION=plain # (or login)
+# Email
+SMTP_ADDRESS=smtp
+SMTP_DOMAIN=smtp.example.com
+SMTP_PORT=1025
 SMTP_USER_NAME=YOUR_EMAIL_ADDRESS
 SMTP_PASSWORD=YOUR_PASSWORD
+SMTP_ENABLE_STARTTLS_AUTO=true
+
+# Setup
+REDMINE_LANG=en
+REDMINE_EMAIL=gtt@example.com
+
+# System
+RAILS_ENV=production
+RAILS_LOG_TO_STDOUT=1
+RAILS_MEMORY_LIMIT=1024
```

Comment:
`puma.rb` file's `RAILS_MEMORY_LIMIT` default value is `2048`, so it's better to change `1024` to `2048`.
https://github.com/gtt-project/docker-gtt/pull/54/files#diff-24409379bdb75ad446bb4e2c18fd4cced1a263b99a3ff96fc3777d8fd8faeab9